config/database.yml

```diff
@@ -58,7 +58,8 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  # TODO: Change database name
+  # TODO: Configure TEST_DATABASE_URL in config/application.yml
+  url:  <%= ENV["TEST_DATABASE_URL"] %>
```

Comment:
@citizen428 this is `DATABASE_URL` across all environments. Maybe we can just remove these lines from `test` like you did above for `development` since you added line 22

Reply:
You're right, I forgot about the `config/application.yml.example` that already includes a `test:` key.